api/tests/prescription/campaign/unit/domain/read-models/CampaignToJoin_test.js

```diff
@@ -109,4 +109,46 @@ describe('Unit | Domain | Models | CampaignToJoin', function () {
       expect(campaignToJoin.isRestricted).to.be.false;
     });
   });
+
+  describe('#setReconciliationFields', function () {
+    it('should return null when no reconcilation defined', function () {
```

Comment:
**Suggestion :** 
``
it('should return null when no reconciliation defined', function () {
``